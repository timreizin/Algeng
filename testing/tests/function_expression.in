a[x]:
x
b[x,y]:
x+y
a[5+4]
a[5*2+(1*2)]
a[-5*4]
c=8
d=5
a[d*c+1-2]
b[c,d]
b[c*d,c-d]
e=a[8]
b[e*c-d,a[4]-5]
