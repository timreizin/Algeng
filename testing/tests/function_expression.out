9
12
-20
39
13
43
58
